src/viewProviders/resources.ts

```diff
@@ -149,36 +146,26 @@ export class ResourceViewProvider implements vscode.TreeDataProvider<ResourceVie
     } else {
       // --- ROOT-LEVEL ITEMS ---
       // NOTE: we end up here when the tree is first loaded
-      const resources: ResourceViewProviderData[] = [];
-
-      // EXPERIMENTAL: check if direct connections are enabled in extension settings
-      const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration();
-      const directConnectionsEnabled: boolean = config.get(ENABLE_DIRECT_CONNECTIONS, false);
```

Comment:
Preview setting check is now done as part of `loadDirectResources()` to clean this section up